lib/converter/rule.go

```diff
@@ -151,25 +152,25 @@ func ruleBackendToAPI(br model.Rule) api.Rule {
 		NotProtocol: br.NotProtocol,
 		NotICMP:     notICMP,
 		Source: api.EntityRule{
-			Tag:         br.SrcTag,
-			Nets:        br.AllSrcNets(),
-			Selector:    br.SrcSelector,
-			Ports:       br.SrcPorts,
-			NotTag:      br.NotSrcTag,
-			NotNets:     br.AllNotSrcNets(),
-			NotSelector: br.NotSrcSelector,
-			NotPorts:    br.NotSrcPorts,
+			Tag:             br.SrcTag,
+			Nets:            br.AllSrcNets(),
+			Selector:        br.SrcSelector,
+			Ports:           br.SrcPorts,
+			NotTag:          br.NotSrcTag,
+			NotNets:         br.AllNotSrcNets(),
+			NotSelector:     br.NotSrcSelector,
+			NotPorts:        br.NotSrcPorts,
 		},
 
 		Destination: api.EntityRule{
-			Tag:         br.DstTag,
-			Nets:        br.AllDstNets(),
-			Selector:    br.DstSelector,
-			Ports:       br.DstPorts,
-			NotTag:      br.NotDstTag,
-			NotNets:     br.AllNotDstNets(),
-			NotSelector: br.NotDstSelector,
-			NotPorts:    br.NotDstPorts,
+			Tag:             br.DstTag,
+			Nets:            br.AllDstNets(),
+			Selector:        br.DstSelector,
+			Ports:           br.DstPorts,
+			NotTag:          br.NotDstTag,
+			NotNets:         br.AllNotDstNets(),
+			NotSelector:     br.NotDstSelector,
+			NotPorts:        br.NotDstPorts,
```

Comment:
looks like this file hasn't actually changed now.  I assume running `goimports` will effectively back this out.